oop-macro/src/lib.rs

```diff
@@ -33,7 +33,7 @@ pub fn extend(attr: TokenStream, mut item: TokenStream) -> TokenStream {
 	let field_name = args.field_name;
 	item.extend::<TokenStream>(
 		quote! {
-			use core::ops::{Deref, DerefMut};
+			use ::core::ops::{Deref, DerefMut};
```

Comment:
still a `use`. i'll fix it

Reply:
I removed it in a later commit, see the updated commit please